pkg/os/util.go

```diff
@@ -113,3 +114,24 @@ func RemoveFileIfExists(path string) error {
 func RunningUsingSSH() bool {
 	return os.Getenv("SSH_TTY") != ""
 }
+
+// Split a multi line string in an array of string, one for each line
+func SplitLines(input string) []string {
```

Comment:
why is this part of crcos (operating system-specific code). this looks like generic line/string manipulation

Reply:
I could not find a more appropriate package, and was not sure adding a `strings` package for 2 functions was worh it. I can make this change if you prefer.

Reply:
I think adding a strings make sense, as I might even think adding the firstline justifies the use. it makes it clearer in intent. perhaps we might even add a getlastline at some point ;-)

Reply:
Done.